railties/lib/rails/generators/rails/app/templates/bin/setup.tt

```diff
@@ -28,12 +28,12 @@ chdir APP_ROOT do
   # end
 
   puts "\n== Preparing database =="
-  system! 'bin/rails db:setup'
+  system! 'rails db:setup'
```

Comment:
I'm definitely torn on this one... I'm again inclined to prefer the safer `bin/rails` (what if they don't have the gem installed globally, and the bundle is vendored?), but on the other hand, it is user-visible. I think safety might trump aesthetics here, though.